doc/reading-list.md

```diff
@@ -86,6 +86,8 @@ Function Shipping
 
 * Function shipping (In-storage-compute) usage in motr: [PDF](PDF/motr_function_shipping.pdf).
 
+  * The layout plan interface allows a generic and fault-tolerant implementation of function shipping: [layout plan](https://github.com/Seagate/cortx-motr/blob/main/layout/layout.h)
```

Comment:
@TechWriter-Mayur , there should be a way to make this a relative (i.e. not absolute) URL.  Can you please fix it to be relative?